src/lib/field.ts

```diff
@@ -908,17 +909,17 @@ class Field {
    * Returns an array of {@link Bool} elements representing [little endian binary representation](https://en.wikipedia.org/wiki/Endianness) of this {@link Field} element.
    *
    * If you use the optional `length` argument, proves that the field element fits in `length` bits.
-   * The `length` has to be between 0 and 255 and the method throws if it isn't.
+   * The `length` has to be between 0 and 254 and the method throws if it isn't.
    *
    * **Warning**: The cost of this operation in a zk proof depends on the `length` you specify,
-   * which by default is 255 bits. Prefer to pass a smaller `length` if possible.
+   * which by default is 254 bits. Prefer to pass a smaller `length` if possible.
    *
    * @param length - the number of bits to fit the element. If the element does not fit in `length` bits, the functions throws an error.
    *
    * @return An array of {@link Bool} element representing little endian binary representation of this {@link Field}.
    */
-  toBits(length?: number) {
-    if (length !== undefined) checkBitLength('Field.toBits()', length);
+  toBits(length: number = 254) {
```

Comment:
we could provide a secure, 255-bits version by treating the bits as a `ForeignField` element and asserting that it's less than the field size. That would add some overhead so it's better to not do it by default. It could be a separate method `toFullBits()` or similar. But not urgent IMO

Reply:
This makes sense. I will implement tomorrow if there is time.